### ISSUE TITLE
`cargo nextest archive` ignores `--frozen`, `--locked`, and `--offline`

### ISSUE THREAD

**Issue by mati865** [nextest-rs/nextest#1327]
Running `cargo nextest archive --help` shows these options among the others:
```
Manifest options:
      --manifest-path <PATH>
          Path to Cargo.toml

      --frozen
          Require Cargo.lock and cache are up to date

      --locked
          Require Cargo.lock is up to date

      --offline
          Run without accessing the network
```
But `--frozen`, `--locked`, and `--offline` are not working (I haven't tested `--manifest-path`).

It can be easily reproduced by removing `Cargo.lock` and running `cargo nextest archive --frozen ...` inside a crate.

**Comment by sunshowers** [comment]
Strange! Definitely a bug. Will investigate soon.

**Comment by sunshowers** [comment]
Found the issue: https://github.com/nextest-rs/nextest/commit/de0d9426e4e09866472da9cf0d87e58bc4ae42b0

The fix will be released as part of cargo-nextest 0.9.69, hopefully in a couple of days. Thanks!
